packages/ui-core/theme/src/lib/components/user-menu/user-menu.component.scss

```
@@ -1,18 +1,29 @@
 @use 'themes' as *;
 
+// The account dropdown is a popup, so it follows the same shadcn rules as the
```

Stylelint (17.14.0)

[error] Unknown word The (CssSyntaxError)

(CssSyntaxError)